src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,8 @@
+package domain
+
+class Board(
+    private val matrix: Matrix,
+) {
+    val dimension: Dimension = matrix.dimension
+    val cells: List<Cell> = matrix.cells.values.toList()
```

Comment:
```suggestion
    val cells: Map<Location, Cell> = matrix.cells
```

Map<Location, Cell> 로 셀을 그려볼까요?